libclc/libspirv/lib/native_cpu/integer/popcount.cl

```diff
@@ -19,7 +19,7 @@ DEF_POPCOUNT_HELPER(char, unsigned char)
 DEF_POPCOUNT_HELPER(short, unsigned short)
 
 #define FUNCTION __spirv_ocl_popcount
-#define __IMPL_FUNCTION
+#define __IMPL_FUNCTION __spirv_ocl_popcount
```

Comment:
this line can be deleted since its value is the same as FUNCTION?

Reply:
this might be a follow-up, not meant to block this PR.

Reply:
Good spot, I think that should work.

I need to push a few more `undef`s anyway so I can do it as part of that.